packages/ui-components/src/components/multi-select-dropdown/multi-select-dropdown.scss

```diff
@@ -23,4 +23,16 @@
 	kv-select {
 		--select-max-height: var(--dropdown-max-height);
 	}
+
+	.selection-clear {
+		@include kv-font-span-regular;
+		user-select: none;
+		color: kv-color('neutral-2');
+		cursor: pointer;
+	
+		&.disabled {
+			color: kv-color('neutral-5');
+			cursor: auto;
```

Comment:
When the clear selection button is disabled, don't we want to show a disallowed cursor?

Reply:
Waiting for confirmation on this from design.

Reply:
Confirmed :)